thumbnail-service/src/main/java/no/sikt/nva/thumbnail/AbstractThumbnailer.java

```diff
@@ -0,0 +1,137 @@
+package no.sikt.nva.thumbnail;
+
+import static no.sikt.nva.thumbnail.ThumbnailerConstants.THUMBNAIL_DEFAULT_HEIGHT;
+import static no.sikt.nva.thumbnail.ThumbnailerConstants.THUMBNAIL_DEFAULT_WIDTH;
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.List;
+import nva.commons.core.JacocoGenerated;
+
+public abstract class AbstractThumbnailer implements Thumbnailer {
+
+    /**
+     * Height of thumbnail picture to create (in Pixel).
+     */
```

Comment:
This comment adds no information that the name does not include already

Reply:
fixed